tests/darwin/cli_functions_test.py

```diff
@@ -86,6 +87,7 @@ def with_verbose_flag(
                     None,
                     None,
                     False,
+                    False,
```

Comment:
More a general comment for this line, but I'd consider annotating (not to confuse things here - not the imagery type!) these properties, e.g.

```
upload_data(
    argumentname=argumentvalue
    [repeated as needed]
)
```

Just for test readability.

Not necessarily for now.  Also looks like a test is needed at some point for the new extract flag you've added, but that these tests don't have them for others so not as part of this task.